gmso/external/convert_foyer_xml.py

```diff
@@ -132,17 +134,21 @@ def _write_gmso_xml(gmso_xml, **kwargs):
     """Given the set of keyword arguments, write a gmso Forcefield xml file."""
     forcefield = etree.Element("ForceField")
 
-    if kwargs.get("name") is not None:
+    if kwargs.get("name"):
```

Comment:
Lets revert this back so we dont get burned by "falsy" values

Reply:
done